### PR TITLE
Fall 24 updates

### PR DESCRIPTION
**Pull request by eiis1000**

### CODE REVIEW

**Comment by kmill** [comment]
I'll take a look again soon, but one solution might be to add a `max-width` to (perhaps) `.desc_img` in `style.css`. Maybe `max-width: 60%` with the appropriate percentage?

You can right click the image and inspect it in the dev tools to see which classes apply.

**Comment by eiis1000** [comment]
All of my attempts seemed to only scale the bounding box of the image (i.e. move the text around it) rather than scale the image itself. The only exception to this was `scale`, which had other undesired effects.

**Comment by kmill** [comment]
Workaround: add something like
```
.desc_img_left .img {
  max-width: 60%
}
```
None of the floating works, but at least the image is an acceptable size.

Edit: in more testing, setting a max-width in px rather than % preserves the floating.

**Comment by eiis1000** [comment]
> Workaround: add something like
> 
> ```
> .desc_img_left .img {
>   max-width: 60%
> }
> ```
> 
> None of the floating works, but at least the image is an acceptable size.
> 
> Edit: in more testing, setting a max-width in px rather than % preserves the floating.

I'm trying this, and it's not actually changing the image, so I must be doing something wrong. Can you make a commit with your proposed change in context? 

Worth noting: I am a backend dev; I have no idea what I'm doing.

**Comment by kmill** [comment]
Oops, I meant `img` not `.img`. In any case, pushed something that seems to work and the image is floating to the left like it should.

**Comment by kmill** [comment]
(I also did a `git merge origin master` and resolved the conflict, so be sure to `git pull`)

File: teptour.js [hunk 1]
```diff
@@ -680,8 +680,9 @@ def_obj("The Foyer", "room", {
   description : `[img 1/foyer/look.jpg left]This is the foyer.
   You can keep going [dir northwest] to the center room.  You
   can see [a subwoofer], [ob 'front desk' 'a desk'],
-  [ob 'colorful lights'], [the mailboxes],
-  and [ob 'foyer mirror' 'large mirror'].`
+  [ob 'colorful lights'], 
+  and [ob 'foyer mirror' 'large mirror']. Something about
```
Comment (kmill) [inline teptour.js]:
```suggestion
  and [ob 'foyer mirror' 'a large mirror']. Something about
```

**Comment by eiis1000** [comment]
Your change fixes it, thank you!

**Comment by eiis1000** [comment]
This is ready to merge - we will make another PR around 2PM "tomorrow" (aka today) with many more updated images/descriptions, which we would veeeeeery much appreciate if you could merge before 3 (when a tour will be starting). (This means you can go to sleep now :) thank you so much for staying up so long!)

**Pull request merged**